supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,6 +10,35 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
 
+[[conscious-language]]
+== Conscious language
+
+The Conscious Language Group is supporting the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects that Red Hat is involved with.
+ 
+.blacklist and whitelist 
+
+When possible, rewrite documentation to avoid these terms.
+
+.Example (before)
```

Comment:
I had a little bit of trouble following these at first glance. It's not initially clear which are grouped together. I'd suggest another way of grouping them somehow.

Could consider having one heading and then using icons to make it clear which is wrong or right. For example:

```
.Example: Removing 'whitelist'
image:images/no.png[no] The following steps demonstrate adding a new rule to whitelist a custom binary.

image:images/yes.png[yes] The following steps demonstrate adding a new rule to allow a custom binary.
```

![image](https://user-images.githubusercontent.com/7443887/130504561-f6439245-75a8-4131-a877-de398827c7e7.png)

Reply:
Great idea. I agree, that will look much more clear. TY